presto-testing/src/main/java/io/prestosql/testing/AbstractTestQueryFramework.java

```diff
@@ -100,8 +104,9 @@ protected abstract QueryRunner createQueryRunner()
 
     @AfterClass(alwaysRun = true)
     public void close()
+            throws IOException
```

Comment:
for better readabillity `afterClassClose`, or something alike, because we also have after-method 'close' in many tests

Reply:
Let me avoid doing refactors in this pull requests.